h/security/policy/__init__.py

```diff
@@ -1,2 +1,2 @@
-from h.security.policy.bearer_token import BearerTokenPolicy
+from h.security.policy.streamer import StreamerPolicy
```

Comment:
The two public classes in `h.security.policy` are now `TopLevelPolicy` (the security policy for the main h app) and `StreamerPolicy` (the security policy for the separate streamer app) which seems right. `BearerTokenPolicy` is now private.